Lesson3/Lesson7/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="/Lesson7/style.css">
+</head>
+<body style="margin:0px;">
```

Comment:
Better not put here

Reply:
Apart from that, do you have any other comments or advice? Maybe you can give me some idea of what to work on? And what else needs to be done in this task?